"Carried over from mozjs60"
Reason: Fix the '-include' directive, otherwise it tries to use the host's header
--- a/js/src/build/js.pc.in
+++ b/js/src/build/js.pc.in
@@ -11,4 +11,4 @@
 # Use -isystem for includes rather than -I, to mark them as "system headers"
 # that don't generate warning diagnostics. For justification, see
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1539036
-Cflags: -include ${includedir}/@JS_LIBRARY_NAME@/js/RequiredDefines.h -isystem ${includedir}/@JS_LIBRARY_NAME@
+Cflags: -include ${pc_sysrootdir}${includedir}/@JS_LIBRARY_NAME@/js/RequiredDefines.h -isystem ${includedir}/@JS_LIBRARY_NAME@
